### PR TITLE
feat: add product and version to databricks sdk

### PR DESCRIPTION
**Pull request by LauJohansson**
This is an implementation of #60 .

### CODE REVIEW

File: src/spetlr/db_auto.py [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 from databricks.sdk import WorkspaceClient
 
+import spetlr.version as spetlrversion
+from spetlr import __name__ as spetlrname
```
Comment (LauJohansson) [inline src/spetlr/db_auto.py:9]: @mrmasterplan - is this a allowed way of importing the spetlrname?

File: src/spetlr/db_auto.py [hunk 2]
```diff
@@ -43,4 +45,6 @@ def getDbApi() -> WorkspaceClient:
     if not host or not token:
         raise Exception("Unable to auto-configure api client.")
 
-    return WorkspaceClient(host=host, token=token)
+    return WorkspaceClient(
+        host=host, token=token, product=spetlrname, product_version=spetlrversion
```
Comment (nfx) [inline src/spetlr/db_auto.py:49]: How does product name look like?.. alphanum and dasges are fine. and I hope the version is SemVer-compatible. If so - lgtm

**Review by nfx — APPROVED** [review]

File: src/spetlr/db_auto.py (hunk 2 above)
Comment (LauJohansson) [reply src/spetlr/db_auto.py:49]: @nfx thanks for your feedback.
productname is alphanum -> it is actually just "spetlr". I just didnt want to hardcode the values - rather import the library name.
The version is SemVer compatible.

File: src/spetlr/db_auto.py (hunk 1 above)
Comment (LauJohansson) [reply src/spetlr/db_auto.py:9]: from spetlr import __version__

**Review by MartinBoge — APPROVED** [review]

**Pull request merged**